webapp/components/notifications/Notification/Notification.vue

```diff
@@ -2,20 +2,29 @@
   <ds-space :class="{ notification: true, read: notification.read }" margin-bottom="x-small">
     <no-ssr>
       <ds-space margin-bottom="x-small">
-        <hc-user :user="post.author" :date-time="post.createdAt" :trunc="35" />
+        <hc-user
+          :user="post.author || comment.author"
+          :date-time="post.createdAt || comment.createdAt"
+          :trunc="35"
+        />
```

Comment:
Maybe:
```vue
        <hc-user v-if="resourceType == 'Post'"
          :user="post.author"
          :date-time="post.createdAt"
          :trunc="35"
        />
        <hc-user v-else
          :user="comment.author"
          :date-time="comment.createdAt"
          :trunc="35"
        />
```

Reply:
Alternatively you could use a computed property:
```js
computed: {
  mentioner() {
    return this.post.id ? this.post.author : this.comment.author
  }
}
```

Reply:
If done the first approach …